app/models/core/IncomeSourceId.scala

```diff
@@ -18,20 +18,21 @@ package models.core
 
 import models.core.IncomeSourceIdHash.mkIncomeSourceIdHash
 
-class IncomeSourceId private(val value: String) extends AnyVal {
+case class IncomeSourceId(value: String) {
+
   def toHash: IncomeSourceIdHash = mkIncomeSourceIdHash(this)
 
   override def toString: String = s"IncomeSourceId: $value"
 }
 
 object IncomeSourceId {
-  def apply(incomeSourceAsString: String): IncomeSourceId = {
-    new IncomeSourceId(incomeSourceAsString)
-  }
-  def mkIncomeSourceId(incomeSourceAsString: String): IncomeSourceId = {
 
-    new IncomeSourceId(incomeSourceAsString)
+//  def apply(incomeSourceAsString: String): IncomeSourceId = {
```

Comment:
Probably don't need this comment anymore

Reply:
think it's gone not sure when but cannot find it anymore